src/js/app.js

```diff
@@ -0,0 +1,196 @@
+"use strict";
+
+angular.module('invertedIndex', [])
+  .filter('range', () => {
+    return function (input, number) {
+      for (let i = 1; parseInt(number) >= i; i++) {
+        input.push(i);
+      }
+      return input;
+    };
+  })
+  .directive('fileUpload', ['alerts', (alerts) => {
+    /*
+     * template to be rendered
+     */
+    const template = `<input type="file" id="file" name="files" multiple>
```

Comment:
Try not to put raw HTML in your js file. Let HTML elements be on html file, you can explore the options of `ng-show, ng-hide or ng-if` to control the display